src/quo2/components/community/community_link_card.cljs

```diff
@@ -0,0 +1,136 @@
+(ns quo2.components.community.community-card
+(:require [quo2.components.markdown.text :as text]
+          [quo2.foundations.colors :as colors]
+          [quo2.components.community.style :as style]
+          [quo2.components.icon :as icons]
+          [react-native.core :as rn]
+          [status-im.i18n.i18n :as i18n]))
+
+
+(defn community-title-and-description [title description image]
+  [rn/view
+   [rn/view
+    {:style {:padding-left    15
+             :flex-direction  :row
+             :align-items     :center}}
+
+    [rn/image
+     {:source image
+      :style {:width  16
+              :height 16}}]
+
+    [rn/view
+     {:style {:margin-left 5}}
+     [text/text {:font-size    15
+                 :font-weight  600
+                 :font-style   :normal
+                 :weight       :semi-bold}
+      title]]]
+
+   [rn/view
+    {:style {:padding-left 15}}
+    [text/text {:font-size       13
+                :font-style      :normal
+                :font-weight     400
+                :letter-spacing  -0.003
+                :weight          :regular}
```

Comment:
Two weights again
And for font-size 13 we use this attribute
`:size` `:paragraph-2`